wiki/People/Community_Contributors/en.md

```diff
@@ -9,102 +8,128 @@ Not to be mistaken as [osu! Alumni](/wiki/People/osu!_Alumni) which are previous
 
 ## 2013
 
-### August
-
-*Also see the forum thread: [osu! community contributors](/community/forums/posts/2489459).*
+<!-- link is dead https://osu.ppy.sh/community/forums/posts/2489459 -->
+*Also see the forum thread: [osu! community contributors](https://web.archive.org/web/20131003162259/https://osu.ppy.sh/forum/p/2489459 "Wayback Machine").*
 
 | User | Contributions |
-| ---- | ------------- |
-| [Darkimmortal](/users/10886) | Developing and maintaining the [osu!record service](http://osu.ppy.sh/forum/t/108092) (replay `.osr` to video file.) |
-| [nanashiRei](/users/807630) | Hosting and maintaining one of osu!'s longest running [beatmap mirrors](https://osu.yas-online.net/). |
-| [RBRat3](/users/307202) | Numerous (beyond count) graphical contributions to the game client and everything else. |
-| [Ballance](/users/165946) | [Achievement](/wiki/Achievements) artwork. |
-| [akrolsmir](/users/576800) | Developing and maintaining [AIBat](/community/forums/topics/55305), the most popular third-party beatmap modding tool ever produced by most people on the community. |
-| [statementreply](/users/126198) | Outstanding performance in bug-fix/feature testing and coding numerous utilities for mappers and modders. |
-| [Loctav](/users/71366) | Organising several major official tournaments (including the OWC). |
-| [Blazevoir](/users/120265) | Insane moderation track record (equivalent to roughly 6 staff members) and almost single-handedly moderating `#osu` during peak hours (9k+ users). |
-| [Piotrekol](/users/304520) | Developing and maintaining [osu!stats](https://osustats.ppy.sh/) and a number of useful utilities for both beatmapping, modding and general play. |
-| [MillhioreF](/users/941094) | Outstanding performance in bugfix/feature testing and handling user support queries. |
-| [DeathxShinigami](/users/49516) | 3+ years of silent work tirelessly handling [beatmap packing](/p/packlist) and several years of managing the beatmap charts. |
-| [LuigiHann](/users/1079) | Years upon years of service, including designing the original beatmap skin, advancing [storyboarding](/wiki/Storyboards) as an art and motivation. |
-| [awp](/users/2650) | Early community management and moderation and huge contributions over the course of his time. |
-| [Sinistro](/users/5530) | Being the first community manager and global moderator, who set the bar extremely high for the [GMT](/wiki/People/Global_Moderation_Team) to follow. |
-| [Ivalset](/users/827) | Being the first staff manager and BATmanager and single-handedly forming an elite beatmap moderation team from nothing. |
+|---|---|
```

Comment:
```suggestion
| --- | --- |
```

Reply:
oh, so it was padded with 1 space on both sides